Real TVS Initial PR (#1746)
pi-base/data: properties/P000238.md

---
uid: P000238
name: Has a real TVS topology
aliases:
  - Topological vector space
  - TVS
refs:
  - wikipedia: Topological_vector_space
    name: Topological vector space on Wikipedia
  - zb: "0867.46001"

>     name: Functional analysis. 2nd ed. (W. Rudin)

Reviewer: ```suggestion
    name: Functional analysis (W. Rudin, 1991)
```
It seems that for other pi-base refs that refer to a later edition, we just add the year after the author's name.
e.g., Engelking

Author: Should we change [S103|P89](https://topology.pi-base.org/spaces/S000103/properties/P000089) while we're at it?

Reviewer: sure

---

> $X$ is homeomorphic to a real topological vector space (TVS).

Reviewer: ```suggestion
$X$ is homeomorphic to a topological vector space (TVS) over $\mathbb R$.
```

> $X$ is homeomorphic to a topological vector space (TVS) over $\mathbb R$.

Reviewer: ```suggestion
$X$ is homeomorphic to a [topological vector space](https://en.wikipedia.org/wiki/Topological_vector_space) (TVS) over $\mathbb R$.
```

> Equivalently, there exists a continuous commutative group operation $(x, y) \mapsto x + y$, and a continuous scalar multiplication operation $\mathbb{R} \times X \to X$, $(\lambda, x) \mapsto \lambda x$, where $\mathbb{R}$ has the Euclidean topology, such that these operations together satisfy the axioms of a real vector space.

Reviewer: ```suggestion
Equivalently, there exists a continuous commutative group operation $(x, y) \mapsto x + y$ on $X$, and a continuous scalar multiplication operation $\mathbb{R} \times X \to X$, $(\lambda, x) \mapsto \lambda x$, where $\mathbb{R}$ has the Euclidean topology, such that these operations together satisfy the axioms of a real vector space.
```

> Many authors, such as Rudin, additionally require separation axioms like {P3}, {P2} or {P1}, though we do not.

Reviewer: Is that really "many authors" or just "some authors"?  The ones I looked at don't assume it.

Author: I just checked a few (Rudin, Zimmer, Kesavan) they all did. Bourbaki does not of course. I checked [TVS on Wikipedia](https://en.wikipedia.org/wiki/Topological_vector_space) and that page coincidentally happens to make almost the same statement:

> Many authors (for example, [Walter Rudin](https://en.wikipedia.org/wiki/Walter_Rudin)), but not this page, require the topology on  X to be T1; it then follows that the space is Hausdorff, and even Tychonoff.

We could make a more neutral claim by instead writing:

> Authors, such as Rudin, may additionally require separation axioms like {P3}, {P2} or {P1}, though we do not.

Reviewer: I asked google AI: "Most modern reference works, such as those by Bourbaki or Trèves, prefer the non-Hausdorff definition for several structural reasons: Generality of Seminorms ... , Quotient Operations ..., Categorical Consistency ....  You are more likely to find Hausdorff required in introductory functional analysis textbooks (like Rudin) or in contexts where unique limits are essential. ..."

So I prefer "Some authors require", but if you want to keep what's currently there, that's fine with me.

Unrelated to that, `separation axioms like {P3}, {P2} or {P1}` is copied from the topological group definition, where T0 does sometimes appear in the definition.  For TVS though, just using `... like {P3} or {P2}` would be enough (i.e., no need to mention the T0 case)?  What do you think?
